site/src/pages/ManagementSettingsPage/OrganizationProvisionersPageView.tsx

```diff
@@ -0,0 +1,40 @@
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+import Button from "@mui/material/Button";
+import type { ProvisionerDaemon } from "api/typesGenerated";
+import { PageHeader, PageHeaderTitle } from "components/PageHeader/PageHeader";
+import { Stack } from "components/Stack/Stack";
+import { Provisioner } from "modules/provisioners/Provisioner";
+import type { FC } from "react";
+import { docs } from "utils/docs";
+
+interface OrganizationProvisionersPageViewProps {
+	provisioners: ProvisionerDaemon[];
+}
+
+export const OrganizationProvisionersPageView: FC<
+	OrganizationProvisionersPageViewProps
+> = ({ provisioners }) => {
+	return (
+		<div>
+			<PageHeader
+				// The deployment settings layout already has padding.
+				css={{ paddingTop: 0 }}
+				actions={
+					<Button
+						endIcon={<OpenInNewIcon />}
+						href={docs("/admin/provisioners")}
+					>
+						Create a provisioner
+					</Button>
+				}
+			>
+				<PageHeaderTitle>Provisioners</PageHeaderTitle>
+			</PageHeader>
+			<Stack spacing={4.5}>
```

Comment:
I feel the way this is defined `gap: spacing * 8,` sort of constrains the UI to consistent spacing in multiples of 8 if we only use whole numbers for spacing. I would argue that for consistency we should keep whole numbers or rethink the spacing * 8 definition.

Reply:
I think we should rethink the `* 8` definition. If you do a search for `spacing={` you'll see that the codebase is already covered in `.5` spacings. 😅 This is just the spacing that the health page already uses, and I think it looks right.